open-machine-learning-jupyter-book/STYLE_GUIDE.md

```diff
@@ -210,6 +210,25 @@ assets/
 
 Note that we may need to set the `width` to `105%` so that all content of `index.html` will be rendered correctly. Also, the `height` has to be set manually.
 
+## Visualization parts
+
+1\. Get the target HTML file, make sure all the bundled files are downloaded and the HTML file can be run separately locally.
+
+2\. Keep only the visualization content, no additional description is needed.
+
+3\. Add the HTML file and bundle to the `open-machine-learning-jupyter-book\assets\html` counterpart.
+
+4\. Embed HTML within the corresponding section. HTML embedding method refer to Python. For example:
+
+```text
```

Comment:
This is Python code. You should use '''python. Please update this in a future PR.